packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Live/Live.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { useSignals, useStore } from 'app/store';
+import { LiveInfo } from './LiveInfo';
+import { LiveButton } from './LiveButton';
+import {
+  Description,
+  ErrorDescription,
+  WorkspaceInputContainer,
+  WorkspaceSubtitle,
+} from '../../elements';
+
+export const Live = observer(() => {
```

Comment:
Can you please move the observer to the end so you create the const and then export it with the observer around it at the bottom?

Reply:
Did it this way to have consistent imports/exports. Moving it to the bottom would require using `export default` or declaring a new variable.

Reply:
hmmm

Im okay with leaving it that way :)